chartscreen.py

```diff
@@ -232,4 +232,19 @@ def create_chart(self, context, image_width, image_height):
                 context.set_source_rgb(color[0], color[1], color[2])
                 context.fill()
 
+                midpoint_angle = angle + slice / 2
+                midpoint_x = x + (r / 2) * math.cos(midpoint_angle)
+                midpoint_y = y + (r / 2) * math.sin(midpoint_angle)
+
+                context.save()
+                context.translate(midpoint_x, midpoint_y)
+                if 0 <= midpoint_angle <= math.pi / 2 or 3 * math.pi / 2 <= midpoint_angle <= 2 * math.pi:
```

Comment:
It'll be great to group these as it's unreadable the way it is.

Reply:
its older.

Reply:
i made it 180 degrees though cannot make it complete straight as there isnt any control i tried it with 0 first then made it completely 180 with slice so that its redable atleast also main reason being sometimes colors are way too close to argue it they differ.